Rubberduck.RibbonDispatcher/Sample Ribbons/RibbonLibrary/src/RibbonCallbacksAdapter.bas

```diff
@@ -0,0 +1,226 @@
+Attribute VB_Name = "RibbonCallbacksAdapter"
+Option Explicit
+
+Public Const ModuleName   As String = "RibbonCallbacksAdapter."
+Public Const DefaultImage As String = "MacroSecurity"
+
+Public Enum ControlSize
+    rdRegular = RdControlSize_rdRegular
+    rdLarge = RdControlSize_rdLarge
+End Enum
+
+Private Function ViewModelFor(ByVal Control As IRibbonControl _
+) As Rubberduck_RibbonDispatcher.RibbonViewModel
+    On Error GoTo EH
+    Dim WkBk As IRibbonWorkbook
```

Comment:
`Dim book As IRibbonWorkbook`  reads better IMO.

Reply:
Having once written code for a library, WkBk has always seemed a better abbreviation to me. In this instance just personal preference I guess.

Reply:
RD inspects for variables that don't contain a vowel, and convention says to start a variable name with a lower case character. `book` is easier for a non English speaker to translate, than `wkBk`.

Naming is hard, but names don't need to be abbreviated.

Reply:
Okay; but it has to be Book rather than book. Being a name that  conceivably might occur in a formal parameter list it must be PascalCased in VBA. Only throw-away names in VBA can tolerate being camelCased, due to the IDE's constant recasing to match that of the last declared instance anywhere in the Project.

Reply:
IME, camelCasing sticks as long as you don't conflict with existing identifiers.

`Dim recordset As ADODB.recordset` isn't a casing problem, it's a naming problem.